### PR TITLE
new playbook for the existing MDTI solution 

### PR DESCRIPTION
**Pull request by Yaniv-Shasha**
1 folder that include azuredeploy.json and a readme file

   Required items, please complete.
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @manishkumar1991 @rahul0216,
Could you please provide your feedback on this.
Thanks

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/AzureDeploy.json [hunk 1]
```diff
@@ -0,0 +1,630 @@
...
+                    }
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "azuresentinel": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('AzureSentinelConnectionName'))]",
+                                "connectionName": "[variables('AzureSentinelConnectionName')]",
+                                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/azuresentinel')]"
+                            }
```
Comment (manishkumar1991) [inline Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/AzureDeploy.json:623]: Azure sentinel connection  should be a "ManagedServiceIdentity" .

for reference kindly refer to :
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/azuredeploy.json

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/AzureDeploy.json [hunk 2]
```diff
@@ -0,0 +1,630 @@
...
+    "variables": {
+        "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]"
+    },
+    "resources": [{
+            "type": "Microsoft.Web/connections",
+            "apiVersion": "2016-06-01",
+            "name": "[variables('AzureSentinelConnectionName')]",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "api": {
```
Comment (manishkumar1991) [inline Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/AzureDeploy.json:51]: Properties of connection should have other details as well , like .

"displayName": "",
"customParameterValues": {},
 "parameterValueType": "Alternative",

**Comment by manishkumar1991** [comment]
Kindly consider standard, Rename the file from AzureDeploy.json to azuredeploy.json .

**Comment by Yaniv-Shasha** [comment]
@manishkumar1991 please review my chanags

**Comment by v-atulyadav** [comment]
Hi @manishkumar1991, please review changes. Thanks

**Comment by manishkumar1991** [comment]
> Hi @manishkumar1991, please review changes. Thanks

Reviewing changes

**Comment by manishkumar1991** [comment]
> Kindly consider standard, Rename the file from AzureDeploy.json to azuredeploy.json .

Hello @Yaniv-Shasha 

Please check on this comment as well

File: Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Prerequisites
+1. This playbook inherits API connections created and established within a base playbook. Ensure you have deployed [MDTI-Base](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Microsoft%20Defender%20Threat%20Intellingence/Playbooks/MDTI-Base/azuredeploy.json) this playbook. If you have trouble accessing your account or your credentials contact your account representative or reach to discussMDTI[@]microsoft.com.
+2. This playbook requires "Microsoft Sentinel Contributor" role to update Incidents.
+
+## Deployment
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FMicrosoft%2520Defender%2520Threat%2520Intelligence%2FPlaybooks%2FMDTI-Data-Cookies%2Fazuredeploy.json" target="_blank">
```
Comment (manishkumar1991) [inline Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/readme.md:12]: link mention in "Deploy to azure " button has azuredeploy.json but file name has upper case, kindly change the playbook arm template name from 

AzureDeploy.json to azuredeploy.json

Comment (Yaniv-Shasha) [reply Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/readme.md:12]: changed it @manishkumar1991

**Comment by manishkumar1991** [comment]
@Yaniv-Shasha , We are seeing that this playbook requires MDTI-base playbook to be installed as a pre requisite .

But MDTI-base playbook, creating a new key vault resource, which is not allowed, as currently our solution packaging doesn't support it , so requesting you to change it and try to use the existing key vault resources .

Kindly refer to the playbook provided below.

https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL/azuredeploy.json

**Comment by v-atulyadav** [comment]
Hi @Yaniv-Shasha, please check comment from @manishkumar1991. Thanks

**Review by manishkumar1991 — APPROVED** [review]
Changes approved.

**Pull request merged**